Tilix4/OpenPype
# Feature: layout compositing nodegroups for characters

Setup compositing nodetree by character, from base nodegroup `nodegroupMatteColorCorrect`.

## Testing notes:
1. Build layout
2. Open `Compositing`, run `arrange nodes` (enable addon first)
3. Add `Viewer node` and connect the same node output as the `Composite` node
4. render F12
5. enter any group (TAB)
6. Unmute (M) any correction node and change parameters, it should alter the render.
